### DISCUSSION TITLE
Error: Rate limit exceeded

### DISCUSSION THREAD

**Discussion by clopmz** [telekom-security/tpotce#1159] [General]
Hi all,

After change passwd for tsec user, my tpot service can not start. Actual error is:

```
HONEY_UUID=40849d5b-3411-4139-9216-92ec1a767e2f
MY_EXTIP=X.X.X.X
MY_EXTIP_LAT={
  "status": 429
  "error": {
    "title": "Rate limit exceeded"
    "message": "You've hit the daily limit for the unauthenticated API.  Create an API access token by signing up to get 50k req/month."
  }
}
MY_EXTIP_LONG={

  "error": {

    "message": "You've hit the daily limit for the unauthenticated API.  Create an API access token by signing up to get 50k req/month."
  }
}
MY_INTIP=172.22.55.8
MY_HOSTNAME=devotedmove
```

 Do I need to reconfigure something?

**Comment by t3chn0m4g3** [comment]
That message is unrelated to the password change, but for some reason you exceeded the limit for [ipinfo](https://github.com/telekom-security/tpotce/blob/1122d3728ee9b1dd921f4cb58e69ea76de0441d3/bin/updateip.sh#L15).

**Reply by clopmz** [reply]
Uhmm ... How can I debug this?

root@devotedmove:/data/nginx/log]# dps.sh
[ ========| System |======== ]
     DATE:  Fri 19 Aug 2022 02:29:47 PM UTC
   UPTIME:   14:29:47 up  2:28,  1 user,  load average: 0.27, 0.59, 0.30
    T-POT:  INACTIVE
BLACKHOLE:  DISABLED

NAME                  STATUS                       PORTS
elasticsearch   Up 2 minutes (healthy)       127.0.0.1:64298->9200/tcp
map_redis       Up 2 minutes
map_web         Exited (137) 2 minutes ago
nginx           Exited (0) 2 minutes ago
spiderfoot      Up 2 minutes (healthy)       127.0.0.1:64303->8080/tcp
kibana                DOWN
logstash              DOWN
map_data              DOWN

**Reply by saad1039** [reply]
I had the same issue. did you figure it out?

**Reply by R3dHash** [reply]
If it helps I applied this fix which worked. It implies to create an account here at ipinfo.io and retrieve a free token. To turn it into a clean fix one would have to change the hardcoded token as an env variable... 
<img width="868" alt="image" src="https://github.com/telekom-security/tpotce/assets/97284989/aa2ee8bf-4ef1-4b42-ae59-6e8127feb765">
